packages/amazonq/src/extensionShared.ts

```diff
@@ -36,6 +37,22 @@ export async function activateShared(context: vscode.ExtensionContext) {
     const contextPrefix = 'amazonq'
     globals.contextPrefix = 'amazonq.' //todo: disconnect from above line
 
+    // Avoid activation if older toolkit is installed
+    // Amazon Q is only compatible with AWS Toolkit >= 3.0.0
+    // Or AWS Toolkit with a development version. Example: 2.19.0-3413gv
+    const toolkit = vscode.extensions.getExtension(VSCODE_EXTENSION_ID.awstoolkit)
+    if (toolkit) {
+        const toolkitVersion = semver.coerce(toolkit.packageJSON.version)
+        const isDevVersion = toolkit.packageJSON.version.toString().includes('-')
+        if (toolkitVersion && toolkitVersion.major < 3 && !isDevVersion) {
+            await vscode.window.showInformationMessage(
+                `The Amazon Q extension is incompatible with AWS Toolkit versions 2.9 and below. Your AWS Toolkit will be updated to version 3.0.`,
```

Comment:
will this string need to be updated to the latest toolkit version prior to release?

Reply:
Also, this makes it sound like we are installing the extension update for them. Is this possible?